README.md

```diff
@@ -1,9 +1,47 @@
 # ansible-documentation
-Clone of ansible/ansible with non-docs content removed
 
-Before you build the docs locally, you need to:
+This repository holds the ReStructuredText (RST) source, and other files, for user documentation related to the Ansible package and Ansible core.
 
-```text
-python3 docs/bin/clone-core.py
-```
+> Documentation for modules and plugins that are officially supported by the Ansible core engineering team is available in the [`ansible/ansible`](https://github.com/ansible/ansible) repository.
 
+## Building Ansible community documentation
+
+Complete the following steps to build Ansible community documentation from this repository:
+
+* Optionally create a Python virtual environment.
+
+    ```bash
+    # Create a virtual environment in your current directory.
+    python3 -m venv venv
+
+    # Activate the virtual environment to start using it.
+    source venv/bin/activate
+    ```
+
+1. Set up your environment.
+
+    ```bash
+    #
+    python3 -m pip install --upgrade pip setuptools six wheel
+
+    python3 docs/bin/clone-core.py
+    ```
+
+2. Install required Python packages.
+
+    ```bash
+    #
+    python3 -m pip install -r requirements.txt
```

Comment:
This file doesn't exist.

Reply:
I think I'm losing it today and should really review my own PRs. Some of the comments and other text I added weren't in the commit (probably b/c I didn't save in the editor).

At risk of further embarrassing myself I'd like to ask what you mean? When I do `clone-core.py` it adds https://github.com/ansible/ansible/blob/devel/requirements.txt to the source tree.

Reply:
I meant that it doesn't exist in this repository. It is, as you mentioned, copied in by the `clone-core.py` script. 

However, AFAIK none of the requirements in that file are used by the docs build or tests in any way (aside from its existence being required to build an sdist of ansible-core from this repo, something that will hopefully not be required in the future).